openedx/openedx-platform
# feat  add drf spectacular 2

How to test:
- `make dev.shell.cms` >> `make requirements`
- `make dev.restart-container.cms`
- open `localhost:18010/cms-api/schema` - should download a swaggerfile
- open `localhost:18010/cms-api/ui` - should show a swagger UI that lists every endpoint of the new studio content API, but nothing else.
